libs/pika/async_mpi/src/mpi_polling.cpp

```diff
@@ -221,17 +263,20 @@ namespace pika::mpi::experimental {
 
             // Eagerly check if request already completed. If it did, call the
             // callback immediately.
+#ifdef EAGER_CHECK
```

Comment:
```suggestion
#ifdef PIKA_MPI_EAGER_CHECK
```
? Or if this is just for debugging please remove it.